sn_networking/src/record_store.rs

```diff
@@ -36,11 +37,8 @@ pub const REPLICATION_INTERVAL_LOWER_BOUND: Duration = Duration::from_secs(180);
 /// Max number of records a node can store
 const MAX_RECORDS_COUNT: usize = 2048;
 
-/// How frequenly we prune the records in the store
-/// eg, if 100 here, every 100 valid PUTs we'll prune the store
-/// This should give us some margin about storing data for nodes that are no longer our responsibility
-/// And may eventually feed into price discovery / storage costs and give us a buffer there too
-const PRUNE_ONCE_EVERY_PUTS: usize = 100;
+/// ~Number of puts per price step
```

Comment:
What does the tilde signify here?

Reply:
Approximate (as there's not necessarily an exact number of puts in the final step eg)